syntax = "proto3";

package apex_datatype;

message Nothing{}

message NDarray{
    bytes data=1;
}

message ListNDarray{
    repeated bytes data=1;
}

message ExpsAndPrios{
    repeated bytes data=1;
    bytes prios=2;
}
